test/data/core/jquery-iterability-transpiled.html

```diff
@@ -3,7 +3,7 @@
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 	<title>jQuery objects transpiled iterability test page</title>
-	<script src="../../../external/core-js-bundle/core-js-bundle.js"></script>
+	<script src="../../../node_modules/core-js-bundle/minified.js"></script>
```

Comment:
[IMPORTANT] We cannot do this. I've actually made a similar changes some time ago and I had to back them out. We do not copy files from `node_modules` to TestSwarm, doing that is a security risk. We have to limit the included files to just the ones we need which means we do need the `external` folder and something similar to `grunt-npmcopy`.

Most of this commit needs to be reversed and we need a pure JS implementation of the `npmcopy` Grunt task.

Since further commits build upon this one, we need to audit the final state of the PR to make sure no `node_modules` references were left in in files accessed by TestSwarm.

Reply:
Ok, I didn't know the Testswarm restriction. Unfortunate, but shouldn't take me long to make an npmcopy equivalent.

Reply:
New npmcopy script is ready now.